### PR TITLE
fix: Update docs for support of arbitrary values

### PR DESCRIPTION
**Pull request by felicia-haggqvist**
This PR updates the CSS docs for the CSS properties regarding the support for arbitrary values and solves JIRA-ticket [WARP-404](https://nmp-jira.atlassian.net/browse/WARP-404), as well as, updates the CSS docs from implementations made in these JIRA-tickets: [WARP-411](https://nmp-jira.atlassian.net/browse/WARP-411), [WARP-412](https://nmp-jira.atlassian.net/browse/WARP-412), [WARP-413](https://nmp-jira.atlassian.net/browse/WARP-413), [WARP-415](https://nmp-jira.atlassian.net/browse/WARP-415).

**What has been done in this PR:**
- Removed the unsupported arbitrary values warning-text from the documentation for `width`, `height`, `min-height`, `min-width`, `max-height`, `max-width`, `grid-template-rows`, `grid-template-cols`, `border-radius`, `content`, `flex-basis`, `flex-grow`, `flex-shrink`, `gap`, `grid-auto-rows`, `order`, `padding` and `translate`.

- Removed unsupported `grid-cols-[{col-spec}]` + `grid-rows-[{row-spec}]` warning-text.

- Added missing `content-empty` to content documentation + removed warning of content-[str] being unsupported.

- Removed the unsupported warning-text to the documentation for `columns`. 

- Added support text for arbitrary values for `background-color`, `border-color`, `border-width`, `grid-auto-columns`,  `outline-color` and `text-color`.

- Updated `@warp-ds/uno` to latest version 

**Currently these CSS properties now support arbitrary values:**
- Border-radius
- Flex-basis
- Flex-grow
- Flex-shrink
- Gap
- Grid-auto-rows
- Grid-auto-cols
- Order
- Translate
- Padding
- Margin
- Columns
- Divide-color
- Top-right-bottom-left
- Transition-timing
- Background-position
- Text-color
- Border-color
- Background-image
- Background-color
- Flex
- Grid-template-columns
- Grid-template-rows
- Height
- Width
- Line-height
- Max-height
- Max-width
- Min-height
- Min-width
- Content

**NOTE**: I have only tested CSS-properties where we have already documented that we support/do not support arbitrary values. There could be more CSS-properties that support/do not support arbitrary values but that just don't have it documented yet.

### CODE REVIEW

File: docs/margin.md [hunk 1]
```diff
@@ -138,14 +138,10 @@ You can also use variant modifiers to target media queries like responsive break
 ```
 
 ### Arbitrary values
-If you need to use a one-off margin value that doesn’t make sense to include in your theme, use square brackets to generate a property on the fly using any arbitrary value.
+If you need to use a one-off margin value that doesn’t make sense to include in your theme, use square brackets to generate a property on the fly using any arbitrary value. Make sure to specify the [CSS unit](https://www.w3schools.com/cssref/css_units.php)
```
[the remark this answers is not in the document]
Comment (felicia-haggqvist) [reply docs/margin.md:141]: Sure I can check if MDN has a link, and if so use that link instead. :)

**Review by imprashast — APPROVED** [review]

File: docs/border-color.md [hunk 2]
```diff
@@ -116,3 +116,11 @@ For example, use `md:s-border-positive` to apply the `s-border-positive` utility
 ```html
 <div class="s-border md:s-border-positive ..."></div>
 ```
+### Arbitrary values
+If you need to use a one-off `border-{color}` value, use square brackets to generate a property on the fly using any arbitrary value. However, we recommend only using our semantic color tokens.
```
Comment (magnuh) [inline docs/border-color.md:120]: Maybe "semantic color tokens" here should be linked to the docs for this? ...and it should probably be "semantic color **classes**", right? 😄

Comment (magnuh) [reply docs/border-color.md:120]: Oh. Sorry. I guess it was to say to not use hexcodes or color names as arbitrary value? 😅 🙈

Comment (magnuh) [reply docs/border-color.md:120]: ...then it is correct. 😄

Comment (felicia-haggqvist) [reply docs/border-color.md:120]: ah ok! So it should stay as "semantic color tokens"? 😄

File: docs/divide-color.md
Comment (magnuh) [inline docs/divide-color.md:1]: There are some bad examples in this file using `divide-teal-400` and such... could we fix those now as well? 😬 😅

Comment (felicia-haggqvist) [reply docs/divide-color.md:1]: We've decided to fix this in a separate task 👯

**Review by magnuh — APPROVED** [review]

**Pull request merged**